config/default/manager_image_patch.yaml

```diff
@@ -8,5 +8,5 @@ spec:
     spec:
       containers:
       # Change the value of image field below to your controller image URL
-      - image: gcr.io/k8s-staging-cluster-api-azure/cluster-api-azure-controller:latest
+      - image: gcr.io/tkg-clc-dev/cluster-api-azure-controller-amd64:dev
```

Comment:
Was this supposed to be changed?

Reply:
Nope. Good catch. Wondering how it got in. Let me fix that. Thank you.